x-pack/legacy/plugins/ml/server/models/data_recognizer/modules/apm_nodejs/ml/datafeed_abnormal_trace_durations_nodejs.json

```diff
@@ -0,0 +1,29 @@
+{
+  "job_id": "JOB_ID",
+  "indices": [
+    "INDEX_PATTERN_NAME"
+  ],
+  "max_empty_searches": 10,
+  "query": {
+    "bool": {
+      "must_not": [
+        {
+          "exists": {
+            "field": "parent.id"
+          }
+        }
+      ],
+      "must": [
+        {
```

Comment:
As above, we normally condense these into a single line of JSON to improve readability.

Reply:
https://github.com/elastic/kibana/pull/53792/commits/f40d0a205dc8e20394e6900b6f98946d857a5b69